gir-files/GstMpegts-1.0.gir

```diff
@@ -108,9 +108,11 @@ Consult the relevant specifications for more details.</doc>
             c:symbol-prefix="atsc_eit">
       <doc xml:space="preserve">Event Information Table (ATSC)</doc>
       <field name="source_id" writable="1">
+        <doc xml:space="preserve">The source id</doc>
```

Comment:
Should probably regenerate the gir with autotools. Meson adds a lot of docs in there and now this one is different than the other .gir files :)

Reply:
Or fix the Meson build to do things as you want? I'm not sure I want to get involved in using Autotools again, especially with the Meson build available and the future of the GStreamer build.

Reply:
I'll update that part, no worries :)

Reply:
If you can, sure. Otherwise I'll do that :)